core/templates/mathjaxConfig.ts

```diff
@@ -2,23 +2,8 @@ window.MathJax = {
   // Need this because technically the type error thrown here is correct.
   // This is not the type of MathJax but MathJaxConfig.
   // @ts-ignore
-  skipStartupTypeset: true,
-  messageStyle: 'none',
-  jax: ['input/TeX', 'output/SVG'],
-  extensions: ['tex2jax.js', 'MathMenu.js', 'MathZoom.js'],
-  showMathMenu: false,
-  showProcessingMessages: false,
-  SVG: {
-    useGlobalCache: false,
-    linebreaks: {
-      automatic: true,
-      width: '500px'
-    },
-    scale: 91,
-    showMathMenu: false,
-    useFontCache: false
-  },
-  TeX: {
-    extensions: ['AMSmath.js', 'AMSsymbols.js', 'autoload-all.js']
+  svg: {
+    scale: .91,
```

Comment:
This is now .91 but it used to be 91. Just to check, that is correct, right?

Can you also point to a list of changes between the old and new mathjax, and confirm that each issue there has been tackled successfully or that it causes no problems?

Reply:
reverted this change